src/Service.Host/client/src/components/parts/PartTemplatesSearch.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { LinkButton, Dropdown } from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Page from '../../containers/Page';
+
+function PartTemplateSearch({ privileges, partTemplates }) {
+    const [template, setTemplate] = useState();
+
+    const createUrl = () => {
+        return `/inventory/part-templates/${template}`;
+    };
+    const canCreate = () => {
+        if (!(privileges.length < 1)) {
+            return privileges.some(priv => priv === 'part.admin');
```

Comment:
I'd stick a link on the Resource to /create if the user has the correct permissions and then use the presence or absence of that link to decide canCreate(). This works too obviously but we're trying to move more towards the HATEOAS approach as Richard said